neon2rvv.h

```diff
@@ -3846,127 +3845,127 @@ FORCE_INLINE uint64x2_t vrsraq_n_u64(uint64x2_t __a, uint64x2_t __b, const int _
 }
 
 FORCE_INLINE int8x8_t vsri_n_s8(int8x8_t __a, int8x8_t __b, const int __c) {
-  uint8_t mask = UINT8_MAX >> __c;
+  uint8_t mask = (__c == 8) ? 0 : UINT8_MAX >> __c;
```

Comment:
how about using
```diff
+uint8_t mask = (uint64_t) UINT8_MAX >> __c;
-uint8_t mask = (__c == 8) ? 0 : UINT8_MAX >> __c;
```

I think in this way we may avoid branching?

Reply:
Hmm except the cast won't work for the 64-bit data types, better to keep things uniform between all implementations?

Reply:
Umm I feel ok for this part, because there are already some exception happened in 64 bits cases. Therefore if the feedback is good enough I am willing to change this. 
How do you think about it? If you think this is unnecessary then I will directly merge this PR

Reply:
Okay, made the change for 8/16/32 versions.